com.outerwildsmods.owmods_cli.yml

```diff
@@ -0,0 +1,47 @@
+app-id: com.outerwildsmods.owmods_cli
+
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.mono6
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+command: owmods
+
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+
+finish-args:
+  - --share=network
+  - --filesystem=/tmp:rw
+  - --filesystem=~/.steam/steam/steamapps/common/Outer Wilds/:rw
+
+modules:
+  - name: Mono
+    buildsystem: simple
+    build-commands:
+      - bash /usr/lib/sdk/mono6/install.sh
+      - cp -ar /usr/lib/sdk/mono6/lib/mono/gac/System.Windows.Forms /app/lib/mono/gac
+  - name: owmods_cli
```

Comment:
why not packaging the gui version instead? CLI aren't really an ideal use case for flathub